frontend/templates/troubleshooting/index.tsx

```diff
@@ -110,10 +100,13 @@ const Wiki: NextPageWithLayout<{
                flexShrink="1"
                id="main"
             >
-               <Head>
-                  {metadata}
-                  <HreflangUrls urls={wikiData.hreflangUrls} />
-               </Head>
+               <Metadata
+                  metaDescription={metaDescription}
+                  metaKeywords={metaKeywords}
+                  canonicalUrl={canonicalUrl}
+                  title={title}
+               />
+               <HreflangUrls urls={wikiData.hreflangUrls} />
```

Comment:
What makes hreflang urls not be part of metadata?

Reply:
Our intent was to have many small components with a single responsibility. The metadata was originally on `<meta>` tags. I personally am fine keeping these three separate. (TagManager, Metadata, HrefLangs), but if you have opinions please voice them.

Reply:
Not a big deal, primarily I was just curious why they were designed this way. I just thought these components are so simple that they could all fit into a single metadata component. 
I'd be interesting to see if there's enough logic that it'd be useful to share them between templates 🤔